specification/deviceregistry/resource-manager/Microsoft.DeviceRegistry/preview/2023-11-01-preview/asset.json

```diff
@@ -0,0 +1,663 @@
+    }
+  },
+  "definitions": {
+    "AssetResponseList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Asset"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "description": "List of Assets."
+    },
+    "Asset": {
+      "type": "object",
+      "required": [
+        "extendedLocation"
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "required": [
+            "assetEndpointProfileUri"
+          ],
+          "allOf": [
+            {
+              "$ref": "#/definitions/AssetProperties"
+            }
+          ],
+          "description": "Asset resource properties."
+        },
+        "extendedLocation": {
+          "type": "object",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "The extended location type."
+            },
+            "name": {
+              "type": "string",
+              "description": "The extended location name."
+            }
+          },
+          "description": "The extended location."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Asset definition."
+    },
+    "AssetPatchPayload": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        },
+        "properties": {
+          "$ref": "#/definitions/AssetProperties",
+          "description": "Asset resource properties."
+        }
+      },
+      "description": "Asset definition for Patch operation."
+    },
+    "AssetProperties": {
+      "description": "Defines the asset properties.",
+      "type": "object",
+      "properties": {
+        "uuid": {
+          "type": "string",
+          "description": "Globally unique, immutable, non-reusable id.",
+          "readOnly": true
+        },
+        "assetType": {
+          "type": "string",
+          "description": "Resource path to asset type (model) definition."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Enabled/Disabled status of the asset."
+        },
+        "externalAssetId": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "Asset id provided by the customer."
+        },
+        "displayName": {
```

Comment:
FYI, On Bluefin/AIO-DP side, we removed `displayName` as a root property from the ARM payload in favor of `description` & `resourceName`. I'm not sure if we want alignments across AIO services.

Reply:
This is required for DOE experience to have a display name associated with an asset, which allows for display characters that are not supported or allowed to be in part of the resource name.